### DISCUSSION TITLE
Polish language translation

### DISCUSSION THREAD

**Discussion by Tg11tobias** [ppy/osu#26815] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/92517726/0d11817d-8f20-4a3d-80fe-db5e1dd3f03b)
"Automatically focus search text box in mod select" is not translated to polish in first setup wizard

**Comment by WhatDamon** [comment]
https://crowdin.com/project/osu-web

**i think here is the best place to deal with your problem.**

> by the way, this string cannot be translated because it is not in crowdin
> also, ppy has not add any new strings to crowdin for about half a year (until i answer your question), so some of the new strings are still in english, except you modify the game's sources code (for your own)

**Reply by Tg11tobias** [reply]
Well
I can read english just fine. Just wanted to report this. I'm gonna try to reach out to PPY on twitter, try to get some info on it.
thx for your answer

**Reply by Joehuu** [reply, marked as answer]
> by the way, this string cannot be translated because it is not in crowdin

It is, see https://crowdin.com/editor/osu-web/551/en-pl?view=comfortable&filter=basic&value=0#18430. Just not approved yet.

**Reply by Tg11tobias** [reply]
couldn't find it myself, thanks
